.gitignore

```diff
@@ -1,8 +1,10 @@
 node_modules
 .eslintcache
+.vscode
 user-build-config.yml
 *.log
 .idea
 user-config.*
 dev-config.*
 /artifacts/
+/ui/node_modules
```

Comment:
`/ui/node_modules`  still needs to go away.

Reply:
will do that right away